### PR TITLE
Initial rebase off of sideangleside's satellite6 bootstrap

### PR DESCRIPTION
**Pull request by mccun934**
Preserving history as well as removing Satellite 6 specific wording

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,49 +17,47 @@ Network Classic and get it registered to Satellite 6.
 * Make an API to Satellite to create the Foreman Host associated with the user specified Org/Location
```
Comment (ehelms) [inline README.md]: Found one!

File: README.md [hunk 2]
```diff
@@ -17,49 +17,47 @@ Network Classic and get it registered to Satellite 6.
...
 * Install the candlepin consumer RPM
```
Comment (ehelms) [inline README.md]: I think this should be the "Katello consumer RPM"

File: README.md [hunk 3]
```diff
@@ -17,49 +17,47 @@ Network Classic and get it registered to Satellite 6.
...
 * Running rhn-migrate-classic-to-rhsm (with the user provided activation key) to get product certs on a system
-* registering the system to Satellite 6
-* Configuring the system with a proper puppet configuration pointing
-  at a Satellite
+* registering the system to Foreman
+* Configuring the system with a proper puppet configuration pointing at a Satellite
```
Comment (ehelms) [inline README.md]: s/Satellite/Foreman ?

File: README.md [hunk 4]
```diff
@@ -17,49 +17,47 @@ Network Classic and get it registered to Satellite 6.
...
 * Removing/disabling old RHN Classic packages/daemons (rhnsd, osad, etc)
 
 ## System not registered to any Red Hat Systems Management Platform:
 
-* Make an API to Satellite to create the Foreman Host associated with the user specified Org/Location
+* Make an API to Foreman to create the Foreman Host associated with the user specified Org/Location
 * Install the candlepin consumer RPM
```
Comment (ehelms) [inline README.md]: s/candlepin/Katello

File: bootstrap.py [hunk 5]
```diff
@@ -72,12 +72,12 @@ def get_architecture():
...
     print "RELEASE - %s" % RELEASE
     print "MAC - %s" % MAC
-    print "SAT6_FQDN - %s" % options.sat6_fqdn
+    print "foreman_fqdn - %s" % options.foreman_fqdn
     print "LOGIN - %s" % options.login
     print "PASSWORD - %s" % options.password
     print "HOSTGROUP - %s" % options.hostgroup
     print "LOCATION - %s" % options.location
-    print "OPERATINGSYSTEM - %s" % options.operatingsystem
+    print "OPERATINGSYSTEM - %s" % optisons.operatingsystem
```
Comment (ehelms) [inline bootstrap.py]: s/optisons/options

**Comment by ehelms** [comment]
Added some comments on the cleanup commit

File: README.md (hunk 1 above)
Comment (mccun934) [inline README.md]: I think I was thinking this was referring to the Satellite 5 server but after re-reading, that isnt the case, will fix

File: README.md [hunk 6]
```diff
@@ -14,52 +14,50 @@ Network Classic and get it registered to Satellite 6.
...
 * Removing/disabling old RHN Classic packages/daemons (rhnsd, osad, etc)
 
 ## System not registered to any Red Hat Systems Management Platform:
 
-* Make an API to Satellite to create the Foreman Host associated with the user specified Org/Location
-* Install the candlepin consumer RPM
-* Running subscription-manager (with the user provided activation key) to register the system. 
-* Configuring the system with a proper puppet configuration pointing
-  at a Satellite
+* Make an API to Foreman to create the Foreman Host associated with the user specified Org/Location
```
Comment (ehelms) [inline README.md]: Should this "Make an API call" ?

File: README.md [hunk 7]
```diff
@@ -14,52 +14,50 @@ Network Classic and get it registered to Satellite 6.
 ## Red Hat Classic & Satellite 5
 
 * Installing subscription-manager and its pre-reqs (updated yum & openssl)
-* Make an API to Satellite to create the Foreman Host associated with the user specified Org/Location
-* Install the candlepin consumer RPM
-* Running rhn-migrate-classic-to-rhsm (with the user provided activation key) to get product certs on a system
-* registering the system to Satellite 6
-* Configuring the system with a proper puppet configuration pointing
-  at a Satellite
+* Make an API to Katello to create the Foreman Host associated with the user specified Org/Location
```
Comment (ehelms) [inline README.md]: Should this be "Make an API call" ?

File: README.md [hunk 8]
```diff
@@ -1,11 +1,11 @@
-# Satellite 6 Bootstrap Script
-Bootstrap Script for migrating systems to Red Hat Satellite 6
+# Katello Bootstrap Script
```
Comment (ehelms) [inline README.md]: Nitpick -- this is inconsistent with https://github.com/Katello/client-bootstrap/pull/1/commits/b283d2061df10b4872b3fe34733a066fd636870e#diff-cd48c161c09e850299d4a00383422690R474

**Comment by ehelms** [comment]
Couple of comments on the cleanup commit -- after those are resolved ACK for me

**Pull request merged**